app/services/scenes/scenes.ts

```diff
@@ -189,9 +188,7 @@ export class ScenesService extends StatefulService<IScenesState> implements ISce
   }
 
   get scenes(): Scene[] {
-    return this.state.displayOrder.map(id => {
-      return this.getScene(id);
-    });
+    return uniqBy(this.state.displayOrder.map(id => this.getScene(id)), x => x.id);
```

Comment:
That's looks like a bug symptom fix. Can you please describe this bug in the TODO comment before this line and add a task to Asana with steps how to reproduce this issue?

Reply:
I've added it closer to where the bug happens.